# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxFT2FontList.cpp
# Commit: 40fcc0cdc593
# Full Hash: 40fcc0cdc593fa337273631de435a2901ec881f6
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-02-07 04:17:40
# Regressor Bug: 1878286
# File Overlap Count: 2
# Description:
#   Bug 1878286 - Eliminate usage of fontlist::Pointer in IPDL. r=gfx-reviewers,lsalzman
#   
#   No change in behavior.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D200564
# ==============================================================================

diff -r c48ca61bc925 -r 40fcc0cdc593 gfx/thebes/gfxFT2FontList.cpp
--- a/gfx/thebes/gfxFT2FontList.cpp	Tue Feb 06 10:15:29 2024 +0000
+++ b/gfx/thebes/gfxFT2FontList.cpp	Tue Feb 06 10:41:50 2024 +0000
@@ -430,7 +430,7 @@
     gfxPlatformFontList* pfl = gfxPlatformFontList::PlatformFontList();
     fontlist::FontList* sharedFontList = pfl->SharedFontList();
     if (!IsUserFont() && mShmemFace) {
-      mShmemFace->SetCharacterMap(sharedFontList, charmap);  // async
+      mShmemFace->SetCharacterMap(sharedFontList, charmap, mShmemFamily);
       if (TrySetShmemCharacterMap()) {
         setCharMap = false;
       }